src/components/Video/VideoPlayerWidget.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from "react-helmet"

```

Comment:
If we're using `react-helmet` here, we'll need to add that as a dependency in the package.json.

Maybe there's a way to re-work this so that react-helmet is not required. e.g. just using the docs to tell folks to include the external JS files however they see fit. Some folks will already be using Helmet and will do it that way, others might have a different preferred-method for including external assets.